packages/@aws-cdk/region-info/test/__snapshots__/region-info.test.js.snap

```diff
@@ -426,7 +426,7 @@ Object {
     "cdkMetadataResourceAvailable": false,
     "domainSuffix": "amazonaws.com",
     "partition": "aws-us-gov",
-    "s3StaticWebsiteEndpoint": "s3-website.us-gov-west-1.amazonaws.com",
+    "s3StaticWebsiteEndpoint": "s3-website-us-gov-west-1.amazonaws.com",
```

Comment:
was this endpoint incorrect previously?

Reply:
according to https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/using-govcloud-endpoints.html at least